### DISCUSSION TITLE
How to read the force of contact?

### DISCUSSION THREAD

**Discussion by Aristomo** [google-deepmind/mujoco#681] [Asking for Help] [answered]
For example, a ball is standing on the ground. How can I get the value of the support force between the ball and the ground?

**Comment by v-r-a** [comment]
You can follow the links in issue #680.

**Comment by yuvaltassa** [comment, marked as answer]
[mj_contactForce](https://mujoco.readthedocs.io/en/latest/APIreference.html#mj-contactforce)
